shell/platform/tizen/tizen_vsync_waiter.h

```diff
@@ -14,31 +14,42 @@ namespace flutter {
 
 class FlutterTizenEngine;
 
-class TizenVsyncWaiter {
+class TdmClient {
```

Comment:
I don't quite understand well... Why did you create a separate TdmClient?

Reply:
Because we request tdm vblank signal at another thread, and FlutterTizenEngine create, destory TizenVsyncWaiter at platform thread. Somethimes TizenVsyncWaiter is destoryed by platfrom thread, but request vblank thread may be receive  vblank signal callback, we might have a problem with wild Pointers, so  I sparate TdmClient from TizenVsyncWaiter, make create, destory TdmClient in ecore thread.

Reply:
Thanks! Got it :+1: